sys/sys/_iovec.h

```diff
@@ -53,6 +53,9 @@ struct iovec {
 	(iovp)->iov_len = (len);					\
 } while(0)
 #define IOVEC_INIT_C(iovp, base, len)	do {				\
```

Comment:
This should be a separate commit. `(base)` should be assigned to a temporary in case it's an expression with side effects. Is the cast needed now that we have polymorphic builtins?

Reply:
It's also bogus; some (hopefully most?) users of IOVEC_INIT_C pass it a user capability directly, so I can now panic the kernel by calling `getrandom(NULL, 0, 0)`.

Reply:
Good point, I'll take it out of the patch.